packages/transport-grpc/src/send-transaction.test.js

```diff
@@ -6,97 +6,112 @@ import {proposer} from "../../sdk/src/build/build-proposer.js"
 import {payer} from "../../sdk/src/build/build-payer.js"
 import {ref} from "../../sdk/src/build/build-ref.js"
 import {authorizations} from "../../sdk/src/build/build-authorizations.js"
-import {preSendCheck} from "../../sdk/src/build/build-pre-send-check.js"
+import {voucherIntercept} from "../../sdk/src/build/build-voucher-intercept.js"
 import {voucherToTxId} from "../../sdk/src/resolve/voucher.js"
 import {resolve} from "../../sdk/src/resolve/resolve.js"
 import {response as responseADT} from "../../sdk/src/response/response.js"
 
-const jsonToUInt8Array = (json) => {
-    var str = JSON.stringify(json, null, 0);
-    var ret = new Uint8Array(str.length);
-    for (var i = 0; i < str.length; i++) {
-        ret[i] = str.charCodeAt(i);
-    }
-    return ret
-};
-
-const hexStrToUInt8Array = (hex) => {
-    return new Uint8Array(hex.match(/.{1,2}/g).map(byte => parseInt(byte, 16)));
-};
-
-const strToUInt8Array = (str) => {
-    var ret = new Uint8Array(str.length);
-    for (var i = 0; i < str.length; i++) {
-        ret[i] = str.charCodeAt(i);
-    }
-    return ret
-};
+const jsonToUInt8Array = json => {
```

Comment:
This happens in multiple files. We should think about consolidating these at some point.